README.md

```diff
@@ -1,2 +1,206 @@
+
+```bash
+psql: could not connect to server: No such file or directory
+Is the server running locally and accepting
+connections on Unix domain socket "/tmp/.s.PGSQL.5432"?
+```
+
+In our case, both are true: the database does not yet exist; hence it is also not running. So in order to create the database, we need to run this command:
+
+```bash
+$ initdb /usr/local/var/postgres -E utf8 -U postgres
+```
+
+Note the `-U postgres` option is specifying that the default user should be `postgres`. Once this process completes, your postgres database will be created, but it is still not running! When you ran this command, however, you should have seen a bunch of output, ending with the following instruction regarding how you can start the database:
+
+```bash
+Success. You can now start the database server using:
+
+    pg_ctl -D /usr/local/var/postgres -l logfile start
+```
+
+So this is our penultimate step, run: `pg_ctl -D /usr/local/var/postgres -l logfile start`. This should start the database running in the background. To test this out, now try to launch the interactive command line with `psql`. Success would mean that you see the following:
+
+```bash
+psql: FATAL:  role "<your username>" does not exist
+```
+
+Despite the error here, if we see this when we run the `psql` command, we are doing well. This error is just stating that there is no user created for our currently logged-in system user account. This is what we'll take care of in Step #3.
+
+For now, what we need to do to finshing things off is to setup postgres to start every time we bootup our machine. To do this we need to do the following:
+
+```bash
+mkdir ~/Library/LaunchAgents    # This directory may already exist, in which case this command is not needed.
+cp /usr/local/Cellar/postgresql/9.5.3/homebrew.mxcl.postgresql.plist ~/Library/LaunchAgents/
+```
+
+What the above does is place the process file that can start the Postgres process into a directory containing other such files that your Mac system will run on startup. To test that this is working, we should now restart our Mac system, log back in, and try the `psql` command again. If things are still working, we should see the error from above.
+
+Okay, almost there. Onward!
+
+**Step #2: Creating Our Root User**
+
+If we've reached this stage, we now have the Postgres database installed and running in the background. We've not yet sucesfully loaded the psql command line interface, but that's what we'll get working now.
+
+Our first step is to see if we can log into the default user account -- `postgres`. To do this, we'll run the `psql` command again, only this time we'll specify the user:
+
+```
+psql -U postgres
+```
+
+This should bring you into a command line context that looks like:
+
+```bash
+psql (9.3.13)
```

Comment:
I'm not sure that it matters, but students might see 9.5.1 -- should there be a note about that?